docker/quickstart.Dockerfile

```diff
@@ -1,35 +1,51 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.3
-
-ENV DEBIAN_FRONTEND=noninteractive
+# TODO(gabrielmbmb): update this `Dockerfile` to multi-staged build to reduce the image size
+FROM argilla/argilla-server:${ARGILLA_VERSION:-latest}
 
 USER root
 
-# Create a directory where Elasticsearch and Argilla will store their data
-# We will use this directory as a volume to persist data between container restarts (mainly in HF spaces)
-RUN mkdir /data
-RUN chown -R elasticsearch:elasticsearch /data
+RUN apt-get update && apt-get install -y \
+    apt-transport-https \
+    gnupg \
+    wget
+
+# Install Elasticsearch signing key
+RUN wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | gpg --dearmor -o /usr/share/keyrings/elasticsearch-keyring.gpg
 
+# Add Elasticsearch repository
+RUN echo "deb [signed-by=/usr/share/keyrings/elasticsearch-keyring.gpg] https://artifacts.elastic.co/packages/8.x/apt stable main" | tee /etc/apt/sources.list.d/elastic-8.x.list
+
+# Copy Argilla distribution files
 COPY scripts/* /
 COPY quickstart.requirements.txt /packages/requirements.txt
 COPY dist/*.whl /packages/
 
-RUN apt update && \
-    apt install -y curl git python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils sudo openssl systemctl && \
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-    python3.9 get-pip.py \
-    # Install Argilla
-    && pip3 install -r /packages/requirements.txt && \
+RUN \
+    # Create an user to run the Argilla server and Elasticsearch
+    useradd -ms /bin/bash argilla && \
+    # Create a directory where Elasticsearch and Argilla will store their data
+    mkdir /data && \
+    # Install Elasticsearch and configure it
+    apt-get update && apt-get install -y elasticsearch=8.8.2 && \
+    mkdir /usr/share/elasticsearch/config && \
+    echo "cluster.name: \"docker-cluster\"\nnetwork.host: 0.0.0.0\npath.data: \"/data/elasticsearch\"bootstrap.system_call_filter: false" > /usr/share/elasticsearch/config/elasticsearch.yml && \
```

Comment:
Not sure if is possible but we could use just the defined ARGILLA_HOME as the parent path for the elasticsearch data. `$ARGILLA_HOME/elasticsearch`

Reply:
Or just bypass the `ARGILLA_HOME` definition and force the value to `/data`

Reply:
some lines below we're doing `ENV ARGILLA_HOME_PATH=/data/argilla`